src/mui/input/ReferenceManyInput.js

```diff
@@ -0,0 +1,235 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import debounce from 'lodash.debounce';
+import Labeled from '../input/Labeled';
+import {
+    crudGetMany as crudGetManyAction,
+    crudGetMatching as crudGetMatchingAction
+} from '../../actions/dataActions';
+import { getPossibleReferences } from '../../reducer/references/possibleValues';
+
+const referenceSource = (resource, source) => `${resource}@${source}`;
+const noFilter = () => true;
+
+/**
+ * An Input component for choosing many reference records. Useful for 'hasMany' relationship.
+ *
+ * @example
+ * {
+ *    postIds: [ "1", "23", "4" ]
```

Comment:
The following example (and the ones below) uses `post_id` and this one `postIds`. Please make them consistent. I think it should be `post_ids` everywhere.